TNO-2935 fix

kkwangsir opened the pull request:
fix originalSubscriber was passed to the `UpdateLinkedSubscription`
fix frontend product missing request after approve one of the users by `handleUserUpdate`

Fosol commented:
@kkwangsir sorry you got this in literally minutes after I have refactored and fixed products.

kkwangsir commented:
> @kkwangsir sorry you got this in literally minutes after I have refactored and fixed products.
> 
> If you pull down the latest code in a separate branch you can test to confirm the issue you were fixing is solved.

Sure. Let me test it. Thanks.
